### ISSUE TITLE
Must the access code be hard coded?

### ISSUE THREAD

**Issue by bwims** [bartbutenaers/node-red-contrib-alexa-notifyme#9]
It would be good if the access code could be supplied via e.g. msg.access

Cutting and pasting the notifyme node between tabs loses the access code.

I would like to keep it in a global variable and cut and paste a change node that feeds it into e.g. msg.access

Any chance of this?

Thanks

Brian

**Comment by bartbutenaers** [comment]
Hi @bwims,

I have created a new branch [msg-access-code](https://github.com/bartbutenaers/node-red-contrib-alexa-notifyme/blob/msg-access-code/README.md) in this repository: the access code is not required anymore in the config screen.  But when not specified in the config screen, then it should be supplied via the input message ```msg.access_code``` field.

Would be nice if you could test this new feature, because I don't use this node myself.  I developed it to help a few guys on the Node-RED forum.  So I could only do a few basic tests myself ...

You can install it directly from Github (when you have git installed), by following command from within your .node-red folder:
```
npm install bartbutenaers/node-red-contrib-alexa-notifyme#msg-access-code
```

Kind regards,
Bart

**Comment by bwims** [comment]
Wow!  I've never seen a request turnaround that quickly before! 

It works perfectly, thanks so much!

Will you be updating the main contrib at some point so I can use the Pallette Manager upgrade facility?

Thanks again,

Brian

**Comment by bartbutenaers** [comment]
Hi Brian,

You are welcome!  When it is such a small change, I rather remove it immediately from my TODO list...
It is now available as version _**1.0.3**_ in your palette:

![image](https://user-images.githubusercontent.com/14224149/127382189-c6d95d87-4aed-4768-86ec-394f103e471e.png)

Bart
